docs-en/chapter_data_structure/number_encoding.md

```diff
@@ -65,7 +65,7 @@ Adding $1$ to the one's complement of negative zero produces a carry, but with `
 
 One last puzzle is the $[-128, 127]$ range for `byte`, with an additional negative number, $-128$. We observe that for the interval $[-127, +127]$, all integers have corresponding sign-magnitude, one's complement, and two's complement, and these can be converted between each other.
```

Comment:
"and these can be converted between each other.
Comment" can be changed to ->
"allowing for mutual conversion between them."

Reply:
Again, I think the original expression is simpler.

Reply:
@YangXuanyi It’s a tradeoff. We prefer authentic expressions, without introducing complicated words.

Obviously, "allowing for mutual conversion between them." is more authentic.